README.md

```diff
@@ -126,6 +127,49 @@ client = Redis.new(semian: {
 })
 ```
 
+#### Net::HTTP
+For the `Net::HTTP` specific Semian adapter, since many external libraries may create
+HTTP connections on the user's behalf, the parameters are instead provided
+by calling specific functions in `Semian::NetHTTP`.
+
+To give Semian parameters, assign a `proc` to `Semian::NetHTTP.raw_semian_options`
+that takes a `semian_identifier` like `http_127_0_0_1_80` or `http_github_com_80`,
+and returns a `Hash` with keys as follows, perhaps in an initialization file.
+
+```ruby
+SEMIAN_PARAMETERS = { tickets: 1,
+                      success_threshold: 1,
+                      error_threshold: 3,
+                      error_timeout: 10 }
+Semian::NetHTTP.raw_semian_options = proc do |semian_identifier|
+  # Let's make it only active for github.com
+  if semian_identifier == "http_github_com_80"
```

Comment:
The other ones are arbitrarily named from their config. We use underscores, but anyone can use anything.

I think this adapter is fundamentally different from the others, and that you want to configure Semian on a host basis. So IMO the config lookup should happen with the domain, not the "identifier". 

I wouldn't even include the port TBH, I can't think of an use case were you'd want different configuration on different ports.

Also I can envision use cases where you want multiple domain to use the same CB, e.g. `customer1.provider.com` and `customer2.provider.com`.

So maybe the collabck should pass the URI, and get the identifier in return. (not 100% sure here, it would require more thoughts, @Sirupsen @fw42 what do you say?)

But that last thing can be addressed in a followup.